### PR TITLE
saving tracks from workflow to jsons

### PR DESCRIPTION
**Pull request by jmyrcha**

### CODE REVIEW

File: EventVisualisation/Workflow/src/FileProducer.cxx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+using namespace std;
+using std::cout;
+using std::endl;
+using std::chrono::duration_cast;
+using std::chrono::milliseconds;
+using std::chrono::seconds;
+using std::chrono::system_clock;
+
+std::deque<std::string> FileProducer::load(std::string path)
```
Comment (shahor02) [inline EventVisualisation/Workflow/src/FileProducer.cxx]: `(const std::string& path)` ?

File: EventVisualisation/Workflow/src/FileProducer.cxx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  for (const auto& entry : std::filesystem::directory_iterator(path)) {
+    if (entry.path().extension() == ".json") {
+      result.push_back(entry.path().filename());
+    }
+  }
+  return result;
+}
+
+FileProducer::FileProducer(std::string path, std::string name, int filesInFolder)
```
Comment (shahor02) [inline EventVisualisation/Workflow/src/FileProducer.cxx]: idem, `const std:string&..`

File: EventVisualisation/Workflow/src/FileProducer.cxx (hunk 1 above)
Comment (jmyrcha) [reply EventVisualisation/Workflow/src/FileProducer.cxx]: ok - done

File: EventVisualisation/Workflow/src/FileProducer.cxx (hunk 2 above)
Comment (jmyrcha) [reply EventVisualisation/Workflow/src/FileProducer.cxx]: ok done

**Comment by shahor02** [comment]
Hi @TimoWilken , the o2 fails with `Fatal error: ...no space left on device'`

**Review by shahor02 — APPROVED** [review]

**Pull request merged**